android/README.md

```diff
@@ -79,7 +79,7 @@ analytics for Debug are associated with an App Bundle ID
       Replace `SERIAL` with the device serial number listed in step 2.
 
 ### Compiling the app's offline help
-Keyman for Android help is maintained in the Markdown files in android/help/.
+Keyman for Android help is maintained in the Markdown files in android/docs/.
```

Comment:
Should we add a note about where Keyman Engine for Android help is maintained? 
The Keyman Engine section is below